temporal-sdk/src/main/java/io/temporal/worker/Worker.java

```diff
@@ -512,11 +515,16 @@ private static SingleWorkerOptions toWorkflowWorkerOptions(
           factoryOptions.getWorkflowHostLocalTaskQueueScheduleToStartTimeout();
     }
 
+    int maxConcurrentWorkflowTaskPollers = options.getMaxConcurrentWorkflowTaskPollers();
```

Comment:
Nit: I'd put this in `WorkerOptions::Builder::validateAndBuildWithDefaults()`...

Reply:
I thought about it and decided against it.
1. Some users may be building with just build. To catch them it will be a duplicate
2. Users may not like validateAndBuildWithDefaults CHANGING their explicitly set values. If they read it somewhere afterward. It's not what this method should do. It should error out if the value specified by the user is incorrect. And we can't error out because of backward compatibility.

After considering it I decided to carry an incorrect but tolerable value over and correct it on a later stage in the Worker.